packages/vue/src/examples/pages/static/Static.vue

```diff
@@ -0,0 +1,163 @@
+            by this privacy policy.
+          </SfTab>
+        </SfTabs>
+      </SfContentPage>
+      <SfContentPage title="Customer service">
+        <SfTabs :open-tab="1">
+          <SfTab title="Size guide">
+            This website ("website") is operated by Luma Inc., which includes
+            Luma stores, and Luma Private Sales. This privacy policy only covers
+            information collected at this website, and does not cover any
+            information collected offline by Luma. All Luma websites are covered
+            by this privacy policy.
+          </SfTab>
+          <SfTab title="Where's my order?">
+            This website ("website") is operated by Luma Inc., which includes
+            Luma stores, and Luma Private Sales. This privacy policy only covers
+            information collected at this website, and does not cover any
+            information collected offline by Luma. All Luma websites are covered
+            by this privacy policy.
+          </SfTab>
+        </SfTabs>
+      </SfContentPage>
+      <SfContentPage title="Store locator">
+        This website ("website") is operated by Luma Inc., which includes Luma
+        stores, and Luma Private Sales. This privacy policy only covers
+        information collected at this website, and does not cover any
+        information collected offline by Luma. All Luma websites are covered by
+        this privacy policy.
+      </SfContentPage>
+    </SfContentPages>
+  </div>
+</template>
+<script>
+import {
+  SfBreadcrumbs,
+  SfContentPages,
+  SfTabs,
+  SfIcon
+} from "../../../../index.js";
+export default {
+  name: "Static",
+  components: {
+    SfBreadcrumbs,
+    SfContentPages,
+    SfTabs,
+    SfIcon
+  },
+  data() {
+    return {
+      activePage: "About us",
+      breadcrumbs: [
+        { text: "Home", route: { link: "#" } },
+        { text: "Help", route: { link: "#" } }
+      ]
+    };
+  },
+  computed: {
+    barTitle() {
+      return this.activePage ? this.activePage : "Home";
+    }
+  },
+  methods: {
+    updateActive(title) {
+      this.activePage = title;
+    }
+  }
+};
+</script>
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+
+@mixin for-desktop {
+  @media screen and (min-width: $desktop-min) {
+    @content;
+  }
+}
+
+#static {
+  box-sizing: border-box;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  @include for-desktop {
+    max-width: 1240px;
+    margin: auto;
+  }
+}
+.breadcrumbs {
+  padding: $spacer-big $spacer-extra-big $spacer-extra-big;
+}
+.static-content {
+  flex: 1;
+}
+.back-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem $spacer-big;
+  background-color: #f1f2f3;
+  &__button {
+    padding: 0;
+    margin: 0;
+    border: 0;
+    background-color: transparent;
+    outline: none;
+    cursor: pointer;
+  }
+  &__title {
+    font-family: $body-font-family-secondary;
+    font-size: $font-size-big-mobile;
+    font-weight: 500;
```

Comment:
```suggestion
    font-weight: $h2-font-weight-desktop;
```

Btw, we really need [all that variables](https://github.com/DivanteLtd/storefront-ui/blob/develop/packages/shared/styles/variables/_typography.scss#L43) for font weight @filrak @przspa ?
Is hard to decide sometimes, why not just `$font-weight-normal` and `$font-weight-title`?

I think almost all of these variables are used only on [`_headings.scss`](https://github.com/DivanteLtd/storefront-ui/blob/develop/packages/shared/styles/global/_headings.scss), but the way it is used (styles directly for Hn tags) could be easily defined by the user with simple CSS globally.

Reply:
we used font-weight 300 (lite), 400 (normal) and 500 (medium)

Reply:
@przspa right, so maybe `$font-weight-muted`, `$font-weight-base` and `$font-weight-title`?
I just thing we don't need one per heading tag and device.
Even to facilitate user customization